modules/navier_stokes/doc/content/syntax/Modules/NavierStokesFV/index.md

```diff
@@ -42,7 +42,23 @@ commonly used in the action syntax is presented below:
 
   !listing include/base/NS.h line=std::string T_fluid
 
-For the default names of other variables used in this action, visit [this site](include/base/NS.h).  
+For the default names of other variables used in this action, visit [this site](include/base/NS.h).
+
+
+## Bernoulli pressure jump treatment
+
```

Comment:
The problem here is that this summarizes something bigger with only its parallel issue.

I would rather you leave the section title but just leave a link to the right page:

```
## Bernoulli pressure jump treatment

Please see [the Bernouilli pressure variable documentation]([BernoulliPressureVariable.md](https://github.com/idaholab/moose/pull/24507/files#diff-e318d2c2747655bc9fde1436532f6abd19c131bf882fc39bde3c2edac2df172c)) for more information.

Reply:
Great idea!